test/functionalTest/cases/0880_timeout_for_forward_and_notifications/forward_that_times_out.test

```diff
@@ -155,13 +155,14 @@ Date: REGEX(.*)
 02. Update/UPDATE E1/T1/A1 on CB (provoking forward to accumulator)
 ===================================================================
 HTTP/1.1 200 OK
-Content-Length: 94
+Content-Length: 147
 Content-Type: application/json
 Date: REGEX(.*)
 
 {
     "errorCode": {
-        "code": "404", 
+        "code": "404",
+        "details": "invalid context provider response",
```

Comment:
I got this as an extra, for 'filling in more info' for returning the text in httpRequestSend ...
Didn't bother to investigate why, as extra info doesn't really hurt